src/Display/Homepage.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useRef } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getPosts } from '../redux/slice/postSlice';
+import Movies from '../components/Details';
+import Header from '../components/Header';
+
+import '../index.css';
+
+const Shows = () => {
+  const posts = useSelector((state) => state.post.post);
+  const dispatch = useDispatch();
+
+  const shouldGetPosts = useRef(true);
+  useEffect(() => {
+    if (shouldGetPosts.current) {
+      shouldGetPosts.current = false;
+      dispatch(getPosts());
+    }
+  }, [dispatch]);
+
+  return (
```

Comment:
- [ ] as the previous CR's already mentioned the design is not matching the example provided by Microverse, please leave a comment in the PR if the requested changes are not clear because I'm not able to explain them in more detail. look at the below examples, some of which the previous CR already mentioned.
- The shape of the arrow is different from that of the actual design as shown
- please remove the margin from the body tag this is also not in the original design.